tests/golden/check-copy-paste/second-file.md

```diff
@@ -0,0 +1,29 @@
+<!--
+ - SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
+ -
+ - SPDX-License-Identifier: MPL-2.0
+ -->
+
+[ First   file  ](./first-file.md)
```

Comment:
I also want a test where such a reference (with link similar to name) will appear between other related references :pray:

Reply:
Done